Framework/Algorithms/src/PolarizationCorrections/HeliumAnalyserEfficiency.cpp

```diff
@@ -32,20 +32,20 @@ using namespace API;
 constexpr double HeliumAnalyserEfficiency::ABSORPTION_CROSS_SECTION_CONSTANT = 0.0733;
 
 namespace PropertyNames {
-auto constexpr INPUT_WORKSPACE{"InputWorkspace"};
-auto constexpr OUTPUT_WORKSPACE{"OutputWorkspace"};
-auto constexpr OUTPUT_FIT_CURVES{"OutputFitCurves"};
-auto constexpr OUTPUT_FIT_PARAMS{"OutputFitParameters"};
-auto constexpr SPIN_STATES{"SpinStates"};
-auto constexpr PD{"GasPressureTimesCellLength"};
-auto constexpr PD_ERROR{"GasPressureTimesCellLengthError"};
-auto constexpr START_LAMBDA{"StartLambda"};
-auto constexpr END_LAMBDA{"EndLambda"};
-auto constexpr IGNORE_FIT_QUALITY_ERROR{"IgnoreFitQualityError"};
-
-auto constexpr GROUP_INPUTS{"Inputs"};
-auto constexpr GROUP_FIT_OPTIONS{"Fit Options"};
-auto constexpr GROUP_OUTPUTS{"Outputs"};
+static const std::string INPUT_WORKSPACE{"InputWorkspace"};
+static const std::string OUTPUT_WORKSPACE{"OutputWorkspace"};
+static const std::string OUTPUT_FIT_CURVES{"OutputFitCurves"};
+static const std::string OUTPUT_FIT_PARAMS{"OutputFitParameters"};
+static const std::string SPIN_STATES{"SpinStates"};
+static const std::string PXD{"PXD"};
+static const std::string PXD_ERROR{"PXDError"};
+static const std::string Start_X{"StartX"};
+static const std::string End_X{"EndX"};
```

Comment:
For consistency with our naming convention for constants in C++, please could the `End_X` variable be re-named to `END_X`?